components/LifeAtTorchbox/AtWorkAtPlay/AtWorkAtPlay.module.scss

```diff
@@ -0,0 +1,143 @@
+.container {
+    max-width: $site-width;
+    margin: 0 auto;
+    padding: 0 $grid;
+    display: flex;
+    flex-direction: column;
+    background: radial-gradient(
+        50% 50% at 50% 50%,
+        var(--color--indigo) 0%,
+        var(--color--dark-indigo) 100%
+    );
+
+    @include media-query(medium) {
+        padding: 0 $grid * 2;
+    }
+
+    @include media-query(large) {
+        flex-direction: row;
+        padding: 0 $grid * 2 $grid * 2 $grid * 2;
+    }
+
+    @include media-query(hero-break) {
+        padding: 0 $grid * 4 $grid * 2 $grid * 4;
+    }
+}
+
+.atWork,
+.atPlay {
+    @include media-query(medium) {
+        max-width: 600px;
+    }
+}
+
+.atWork {
+    @include media-query(large) {
+        max-width: calc(50% - $grid * 2);
+    }
+    @include media-query(hero-break) {
+        max-width: 480px;
+    }
+
+    & .title {
+        margin-top: 0;
+        margin-bottom: 0;
+
+        @include media-query(large) {
+            margin-bottom: $grid;
+            margin-top: 0;
+        }
+    }
+}
+
+.atPlay {
+    position: relative;
+    @include media-query(medium) {
+        align-self: flex-end;
+    }
+
+    @include media-query(large) {
+        align-self: flex-start;
+        width: calc(50% + $grid * 4);
+        margin-left: $grid * 4;
```

Comment:
I'm not sure about using calc here to keep the padding between the flex items. Change this to width 50% and margin-left, margin-right $grid for each element?

Reply:
Yep I'd go with the latter